tests/processor/test_orchestrator.py

```diff
@@ -79,4 +99,18 @@ def test_prepare_no_accounts(self, mock_downloader, mock_accounts_accessor):
         orchestrator = Orchestrator()
         reports = orchestrator.prepare()
 
-        self.assertEqual(reports, [])
+        self.assertEqual(len(reports), 0)
```

Comment:
Is there a reason to change this? They are functionally equivalent.

IMO, the version without `len()` is more readable.

Reply:
I have no idea.  I think this was a merge-conflict resolution issue or something.  I'll go with (reports, [])